server/sheet_organizer.go

```diff
@@ -0,0 +1,62 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/google/jsonapi"
+	"github.com/labstack/echo/v4"
+)
+
+func (s *Server) registerSheetOrganizerRoutes(g *echo.Group) {
+	g.POST("/sheet/:sheetID/organizer", func(c echo.Context) error {
```

Comment:
Shall we follow something similar to BackupSettingUpsert(). Then this should be PATCH instead of POST. And we should only take SheetOrganizerUpsert API struct. Right now we have public Find/Patch/Create which can be confusing to readers.

In other words, this existence if/else switch should be implementation details of the store.